src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.2.0
+
+### New Feature
+
+- Support for self-signed and private CA with Docker
+- Buildkit port is now configurable (default 443)
+- Add option for sharing namespace with users within organization
+- UI pagination support for GitHub branches list
+- Add permanently user deletion command
```

Comment:
is this a Cloud feature? or a self-hosted one?

Reply:
I think we did it for self-hosted. @AdrianPedriza can you confirm?

Reply:
it is a Cloud feature